src/type/s2k/index.js

```diff
@@ -0,0 +1,35 @@
+import defaultConfig from '../../config';
+import Argon2S2k, { Argon2OutOfMemoryError } from './argon2';
+import OtherS2K from './generic';
```

Comment:
... and here as well (renaming a class on import makes it quite confusing)

Reply:
Yeah, I just didn't settle on the name 😅